src/github/pullRequestManager.ts

```diff
@@ -748,3 +894,36 @@ export function getEventType(text: string) {
 			return EventType.Other;
 	}
 }
+
+const ownedByMe: Predicate<GitHubRepository> = repo => {
+	const { currentUser=null } = repo.octokit as any;
+	return repo.remote.owner === currentUser.login;

```

Comment:
let's check that `currentUser` exists before dereferencing `login`

Reply:
done